### PR TITLE
Experimental features

### PR DESCRIPTION
**Pull request by achrysaetos**
**ABOUT**
These features will just be deployed to beta for the time being.
This branch will keep track of linked issues and provide occasional updates.

**UPDATES**
Sorting, searching, and filtering functionalities complete.

Screen Shot 2021-03-30 at 6 31 04 PM (2)

### CODE REVIEW

File: frontend/src/components/EvaluationResponses.js [hunk 1]
```diff
@@ -55,171 +78,317 @@ const EvaluationResponses = ({ crn, info }) => {
...
+    });
+    const uniques = array.filter((value) => {
+      return ++frequency[value] === 1;
+    });
+    return uniques.sort((a, b) => {
+      return frequency[b] - frequency[a];
+    });
+  };
+
+  const summarize_comments = useRef(null);
```
Comment (ErikBoesen) [inline frontend/src/components/EvaluationResponses.js:103]: Use camelCase to match standard in this file

File: frontend/src/components/EvaluationResponses.js [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+
 import styles from './EvaluationResponses.module.css';
 import { TextComponent } from './StyledComponents';
+import { evalsColormap } from '../queries/Constants';
+
+// Set up parameters for natural
+const language = 'EN';
+const defaultCategory = 'N';
+const defaultCategoryCapitalized = 'NNP';
+const lexicon = new natural.Lexicon(
```
Comment (ErikBoesen) [inline frontend/src/components/EvaluationResponses.js:14]: It strikes me that lexical analysis of evaluations is probably something we'd want to split out to a separate file.

File: frontend/src/components/EvaluationResponses.js [hunk 3]
```diff
@@ -55,171 +78,317 @@ const EvaluationResponses = ({ crn, info }) => {
...
+          }
+          if (k === 'knowledge/skills') {
+            setDataDefault2(skills_comments.current);
+          }
+          if (k === 'strengths/weaknesses') {
+            setDataDefault2(strengths_comments.current);
+          }
+          if (k === 'summary') {
+            setDataDefault2(summarize_comments.current);
+          }
```
Comment (ErikBoesen) [inline frontend/src/components/EvaluationResponses.js:320]: Perhaps a switch/case statement?

File: frontend/src/components/EvaluationResponses.js (hunk 1 above)
Comment (achrysaetos) [reply frontend/src/components/EvaluationResponses.js:103]: Will do.

File: frontend/src/components/EvaluationResponses.js (hunk 2 above)
Comment (achrysaetos) [reply frontend/src/components/EvaluationResponses.js:14]: Hmm, maybe. I'll pull it out in a new commit later if it's necessary. Thanks for the feedback.

**Pull request merged**